app/controllers/chargeback_controller.rb

```diff
@@ -573,18 +685,38 @@ def cb_rate_get_form_vars
     @sb[:rate] = @edit[:rate]
     @edit[:new][:description] = params[:description] if params[:description]
     @edit[:new][:details].each_with_index do |_detail, i|
-      @edit[:new][:details][i][:rate] = params["rate_#{i}".to_sym] if params["rate_#{i}".to_sym]
       @edit[:new][:details][i][:per_time] = params["per_time_#{i}".to_sym] if params["per_time_#{i}".to_sym]
       @edit[:new][:details][i][:per_unit] = params["per_unit_#{i}".to_sym] if params["per_unit_#{i}".to_sym]
+      # Save tiers into @edit
+      (0..@sb[:num_tiers][i].to_i - 1).each do |j|
```

Comment:
why is this needed?  `0..@sb[:num_tiers][i].to_i - 1`  Why are we indexing into something else?  I don't understand.